corehq/apps/export/forms.py

```diff
@@ -1070,3 +1074,45 @@ def extra_fields(self):
                 data_bind='value: dateRange',
             ),
         ]
+
+
+class DatasourceExportDownloadForm(forms.Form):
+
+    data_source = forms.ChoiceField(
+        label=gettext_lazy("Select project data source"),
+        required=True,
+    )
+
+    def __init__(self, domain, *args, **kwargs):
+        super(DatasourceExportDownloadForm, self).__init__(*args, **kwargs)
+        self.helper = FormHelper()
+        self.helper.label_class = 'col-sm-3 col-md-2 col-lg-2'
+        self.helper.field_class = 'col-sm-9 col-md-8 col-lg-3'
+
+        self.fields['data_source'].choices = self.domain_datasources(domain)
+
+        self.helper.layout = crispy.Layout(
+            crispy.Fieldset(
+                "",
+                crispy.Div(
+                    crispy.Field(
+                        'data_source',
+                        css_class='input-xlarge',
+                        data_bind='value: dataSource'
+                    ),
+                    data_bind='visible: haveDatasources'
+                ),
+            ),
+            hqcrispy.FormActions(
+                StrictButton(
+                    _("Download DET Config file"),
```

Comment:
I'm not confident that all users will know that "DET" stands for the Data Export Tool. What about this? ...
```suggestion
                    _("Download a query file for the Data Export Tool"),
```

Reply:
Good point!